create-turbo/src/shouldUseYarn.ts

```diff
@@ -1,14 +1,10 @@
-import { execSync } from "child_process";
+import { hasExecutable } from "./hasExecutable";
 
-export function shouldUseYarn(): boolean {
```

Comment:
Why is this insufficient?

Reply:
It's not. I just figured, that we can use `hasExecutable` in order to determine whether or not yarn is installed. Since `hasExecutable` returns a `Promise<boolean>` I just removed the type here.